This patch undoes the following Change that was applied to
GCC 4.9 on 2016-02-01:

2016-02-01  John David Anglin  <danglin@gcc.gnu.org>

        PR target/68741
	* inclhack.def (hpux_vsscanf): New fix.
	* fixincl.x: Regenerated.
	* tests/base/stdio.h [HPUX_VSSCANF_CHECK]: New test.

This change causes the fixincludes (at least when building in
the Gentoo emerge system) to generate an extra NULL test at
the end of their list, which causes the overall GCC emerge to
fail.

This patch (which removes the change above) is intended to
be temporary, while that actual bug in the change above is
found and fixed.

diff --git a/fixincludes/fixincl.x b/fixincludes/fixincl.x
index 079c5bf..75df492 100644
--- a/gcc-4.9.2/fixincludes/fixincl.x
+++ b/gcc-4.9.2/fixincludes/fixincl.x
@@ -1,12 +1,12 @@
 /*  -*- buffer-read-only: t -*- vi: set ro:
- * 
+ *
  * DO NOT EDIT THIS FILE   (fixincl.x)
- * 
- * It has been AutoGen-ed  Monday February  1, 2016 at 03:26:15 PM EST
+ *
+ * It has been AutoGen-ed  August 12, 2014 at 05:02:39 PM by AutoGen 5.18
  * From the definitions    inclhack.def
  * and the template file   fixincl
  */
-/* DO NOT SVN-MERGE THIS FILE, EITHER Mon  1 Feb 2016 15:26:15 EST
+/* DO NOT SVN-MERGE THIS FILE, EITHER Tue Aug 12 17:02:39 MSK 2014
  *
  * You must regenerate it.  Use the ./genfixes script.
  *
@@ -15,7 +15,7 @@
  * certain ANSI-incompatible system header files which are fixed to work
  * correctly with ANSI C and placed in a directory that GNU C will search.
  *
- * This file contains 230 fixup descriptions.
+ * This file contains 228 fixup descriptions.
  *
  * See README for more information.
  *
@@ -4084,43 +4084,6 @@ static const char* apzHpux11_VsnprintfPatch[] = {
 
 /* * * * * * * * * * * * * * * * * * * * * * * * * *
  *
- *  Description of Hpux_Vsscanf fix
- */
-tSCC zHpux_VsscanfName[] =
-     "hpux_vsscanf";
-
-/*
- *  File name selection pattern
- */
-tSCC zHpux_VsscanfList[] =
-  "stdio.h\0";
-/*
- *  Machine/OS name selection pattern
- */
-tSCC* apzHpux_VsscanfMachs[] = {
-        "*-*-hpux*",
-        (const char*)NULL };
-
-/*
- *  content selection pattern - do fix if pattern found
- */
-tSCC zHpux_VsscanfSelect0[] =
-       "(extern int vsscanf\\()char";
-
-#define    HPUX_VSSCANF_TEST_CT  1
-static tTestDesc aHpux_VsscanfTests[] = {
-  { TT_EGREP,    zHpux_VsscanfSelect0, (regex_t*)NULL }, };
-
-/*
- *  Fix Command Arguments for Hpux_Vsscanf
- */
-static const char* apzHpux_VsscanfPatch[] = {
-    "format",
-    "%1const char",
-    (char*)NULL };
-
-/* * * * * * * * * * * * * * * * * * * * * * * * * *
- *
  *  Description of Hpux8_Bogus_Inlines fix
  */
 tSCC zHpux8_Bogus_InlinesName[] =
@@ -9405,9 +9368,9 @@ static const char* apzX11_SprintfPatch[] = {
  *
  *  List of all fixes
  */
-#define REGEX_COUNT          269
+#define REGEX_COUNT          267
 #define MACH_LIST_SIZE_LIMIT 187
-#define FIX_COUNT            230
+#define FIX_COUNT            228
 
 /*
  *  Enumerate the fixes
@@ -9507,7 +9470,6 @@ typedef enum {
     HPUX11_SIZE_T_FIXIDX,
     HPUX11_SNPRINTF_FIXIDX,
     HPUX11_VSNPRINTF_FIXIDX,
-    HPUX_VSSCANF_FIXIDX,
     HPUX8_BOGUS_INLINES_FIXIDX,
     HPUX_C99_INTPTR_FIXIDX,
     HPUX_C99_INTTYPES_FIXIDX,
@@ -10115,11 +10077,6 @@ tFixDesc fixDescList[ FIX_COUNT ] = {
      HPUX11_VSNPRINTF_TEST_CT, FD_MACH_ONLY | FD_SUBROUTINE,
      aHpux11_VsnprintfTests,   apzHpux11_VsnprintfPatch, 0 },
 
-  {  zHpux_VsscanfName,    zHpux_VsscanfList,
-     apzHpux_VsscanfMachs,
-     HPUX_VSSCANF_TEST_CT, FD_MACH_ONLY | FD_SUBROUTINE,
-     aHpux_VsscanfTests,   apzHpux_VsscanfPatch, 0 },
-
   {  zHpux8_Bogus_InlinesName,    zHpux8_Bogus_InlinesList,
      apzHpux8_Bogus_InlinesMachs,
      HPUX8_BOGUS_INLINES_TEST_CT, FD_MACH_ONLY,
diff --git a/fixincludes/inclhack.def b/fixincludes/inclhack.def
index defdde5..0a6fb80 100644
--- a/gcc-4.9.2/fixincludes/inclhack.def
+++ b/gcc-4.9.2/fixincludes/inclhack.def
@@ -2172,20 +2172,6 @@ fix = {
 };
 
 /*
- *  Fix missing const in hpux vsscanf declaration
- */
-fix = {
-    hackname = hpux_vsscanf;
-    mach    = "*-*-hpux*";
-    files   = stdio.h;
-    select  = '(extern int vsscanf\()char';
-    c_fix   = format;
-    c_fix_arg = "%1const char";
-
-    test_text = 'extern int vsscanf(char *, const char *, __va_list);';
-};
-
-/*
  *  get rid of bogus inline definitions in HP-UX 8.0
  */
 fix = {
